protocols/tests/test_migration/test_migration_reports_3_0_0_to_participant_1_0_0.py

```diff
@@ -1,22 +1,24 @@
 from unittest import TestCase
 
-from protocols.util import get_valid_cancer_participant_3_0_0
-from protocols.util import get_valid_cancer_participant_1_0_0
 from protocols.reports_3_0_0 import CancerParticipant as CancerParticipant_old
 from protocols.participant_1_0_0 import CancerParticipant as CancerParticipant_new
 from protocols.migration.migration_reports_3_0_0_to_participant_1_0_0 import MigrateReports3ToParticipant1
+from protocols.util.factories.avro_factory import GenericFactoryAvro
+from protocols.util.dependency_manager import VERSION_400, VERSION_300
```

Comment:
Can we please change this line to:
```
from protocols.util.dependency_manager import VERSION_400
from protocols.util.dependency_manager import VERSION_300
```
or 
```
from protocols.util import dependency_manager
```
and refer to `dependency_manager.VERSION_400` and `dependency_manager.VERSION_300`